mysql-test/suite/perfschema/t/crash_tokudb.test

```diff
@@ -0,0 +1,40 @@
+# TokuDB PFS crash test:
+# Make sure FTI instrumentation is on, execute base DDL, DML queries
+# and make sure there is no crash.
+
+--source include/not_embedded.inc
+--source include/have_perfschema.inc
+--source include/have_tokudb.inc
+
+SELECT COUNT(*) > 0 FROM performance_schema.setup_consumers
+  WHERE ENABLED = "NO";
+SELECT COUNT(*) > 0 FROM performance_schema.setup_instruments
+  WHERE NAME LIKE "%/fti/%" AND (ENABLED = 'NO' OR TIMED = "NO");
+
+SET default_storage_engine=TokuDB;
+
+CREATE TABLE t(a INT AUTO_INCREMENT PRIMARY KEY, b INT);
```

Comment:
Why not just use ENGINE=TOKUDB instead of playing around with having to set and reset the default storage engine?